formats/widget/SRF_ListWidget.php

```diff
@@ -37,20 +37,26 @@ public function getName() {
 	protected function getResultText( QueryResult $res, $outputmode ) {
 		// Initialize
 		static $statNr = 0;
-		// $this->isHTML = true;
 
+		$this->hasTemplates = ( $this->params['template'] !== '' );
 		$listType = $this->params[ 'listtype' ] === 'ordered' || $this->params[ 'listtype' ] === 'ol' ? 'ol' : 'ul';
 
 		$builder = new ListResultBuilder( $res, $this->mLinker );
 
 		$builder->set( $this->params );
+
 		$builder->set( [
 			'format' => $listType,
 			'link-first' => $this->mLinkFirst,
 			'link-others' => $this->mLinkOthers,
 			'show-headers' => $this->mShowHeaders,
 		] );
 
+		// this does not seem to work
+		if ( $this->hasTemplates ) {
```

Comment:
Then why are you adding it?

Reply:
I would like to disable `links` when `template` parameter is defined, do you have any hint how to achieve this ?

Reply:
fixed, I've commented the block